packages/react-debug-tools/src/ReactDebugHooks.js

```diff
@@ -260,6 +264,17 @@ function useDeferredValue<T>(value: T, config: TimeoutConfig | null | void): T {
   return value;
 }
 
+function useOpaqueReference(): string | IdObject {
+  const hook = nextHook();
+  const value = hook === null ? 'FAKE_ID' : hook.memoizedState;
```

Comment:
The name  "FAKE_ID" doesn't say much and is kind of confusing if you do end up seeing it. What's Fake about it? Can I search for it? It also doesn't apply as a string when in React Native for example. Let's just leave it as `undefined`  if it's not there.

I believe this can also cause an error if you inspect a hydrated component since `toString` and `valueOf` will throw. We should probably detect if it's a `REACT_OPAQUE_OBJECT_TYPE` and do something special. Perhaps just leaving it as `undefined` as well.